### ISSUE TITLE
How to blacklist certain words/commands?

### ISSUE THREAD

**Issue by rngtng** [bbatsov/rubocop#2939]
We've quite often the problem that someone forgets to [unfocus rspec test](https://relishapp.com/rspec/rspec-core/v/3-4/docs/example-groups/aliasing). To avoid this, I'd like to
fail a rubocop run if e.g. `fdescribe`, `fit` and `fcontext` are found within the code. Is there sth like a Blacklist Cop I could (re-)use for that?

**Comment by alexdowad** [comment]
Writing a custom cop would take less than 15 lines of code.

**Comment by mikegee** [comment]
I think I would prefer the solution you are asking for, but here's how I've solved this so far.

I focus specs such that Rubocop doesn't like the spacing:

```ruby
it 'works',focus:true do
  expect(thing).to work
end
```

I fail make RSpec fail when there are focused specs in CI:

```ruby
if ENV['CI']
  config.before(:example, :focus) { fail 'Should not commit focused specs' }
end
```

from [Myron's suggestion](https://github.com/rspec/rspec-core/issues/1747).

**Comment by rngtng** [comment]
thanks @mikegee  - `config.before(:example, :focus)` does the trick!
